translator/GCP_cloud_function/cloud_function/main.py

```diff
@@ -52,18 +49,17 @@ def parse_xml(xml_string):
 def get_wzdx_schema(schema_file_name):
```

Comment:
Similar to `parse_xml`, this function isn't gaining us much. It is doing a little more with opening the file, then parsing it in json so it may be worth leaving in. If we do leave it in, we need to write unit tests for it to show what happens with a valid file, and what happens if the file doesn't exist.

Reply:
This was added to make unit testing easier

Reply:
added edge case for  get_wzdx_schema and for invalid data i named the function test_get_wzdx_schema_invalid_data() and pushed the updated code to dev